ui-src/src/app/configurations/application-properties/application-properties.component.html

```diff
@@ -0,0 +1,70 @@
+<div class="container">
+  <div [hidden]="submitted">
+    <legend>Application Properties</legend>
+
+    <form (ngSubmit)="onSubmit()" #heroForm="ngForm">
```

Comment:
Are we not worried about all the submit buttons

Reply:
trying to make the components separately first, don't plan to keep them. There will just be one over all submit button at the end.